app/javascript/chat/chat.jsx

```diff
@@ -75,7 +75,7 @@ export default class Chat extends Component {
       activeContent: {},
       fullscreenContent: null,
       videoPath: null,
-      expanded: window.innerWidth > 600,
+      expanded: window.innerWidth > 767,
```

Comment:
`767` seems to be used in a few places. consider making it a constant. It will also make it clearer what that number means which I'm assuming is for low resolution tablets, phones in landscape mode?